client/src/__snapshots__/Home.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render home page 1`] = `ReactWrapper {}`;
```

Comment:
This snapshot doesn't look correct. I think we should be using `-react-test-renderer` instead of `enzyme` to perform snapshot testing. It is the established snapshot testing framework of choice, and it is used in the WaffleMenu tests which I believe is the only one with snapshots at the moment.

Reply:
Ahh yes, I missed that, was referring to the builder for snapshot testing instead. will see how this change affects my previous changes